main.py

```diff
@@ -144,10 +147,44 @@ def check_date(store, date_to_check):
         '{}: {} -> {}'.format(op, last[astart:aend], booked_list[bstart:bend])
         for op, astart, aend, bstart, bend in  difflib.SequenceMatcher(None, last, booked_list).get_opcodes()
         if op != 'equal'
-    ]
+    ] #Contains the difference between the current details and the previous history file.
 
     if diff:
         pprint(diff)
+        send_email(os.environ['EMAIL_HOST'],
+                   int(os.environ['EMAIL_PORT']),
+                   os.environ['EMAIL_USER'],
+                   os.environ['EMAIL_PASSWORD'],
+                   os.environ['EMAIL_FROM'],
+                   os.environ['EMAIL_TO'],
+                   str(diff).encode('utf-8'), #Encoding is necessary due to the en-dash character.
```

Comment:
Isn't this enough?
```python
diff.encode('utf-8')
```

Reply:
No, because diff is a list. I actually tried thought the same at first.

Reply:
in that case:
```python
pprint.pformat(diff).encode('utf-8')
```